### PR TITLE
chore(repo): Clean up workflows

### PR DESCRIPTION
**Pull request by tmilewski**
## Description

It's now caching the deps appropriately.

-  The changesets action is having issues with `&&`, so I moved that part back to NPM scripts.
- Force clean builds
- Only install Playwright Chromium.

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [x] `build/tooling/chore`

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -46,14 +46,14 @@ jobs:
           playwright-enabled: true # Must be present to enable caching on branched workflows
 
       - name: Build
-        run: npx turbo build $TURBO_ARGS
+        run: npx turbo build $TURBO_ARGS --force
```
Comment (tmilewski) [inline .github/workflows/release.yml:49]: Force a clean build

File: .github/actions/init/action.yml [hunk 2]
```diff
@@ -120,4 +120,4 @@ runs:
     - name: Install Playwright Browsers
       if: inputs.playwright-enabled == 'true' && steps.playwright-cache.outputs.cache-hit != 'true'
       shell: bash
-      run: npx playwright install --with-deps
+      run: npx playwright install chromium
```
Comment (tmilewski) [inline .github/actions/init/action.yml:123]: Only install the necessary browsers

**Review by BRKalow — APPROVED** [review]

**Review by BRKalow — APPROVED** [review]
👍

**Review by nikosdouvlis — COMMENTED** [review]
Looks good!
I think we can keep using the cache for canary and snapshot releases

**Pull request merged**